### DISCUSSION TITLE
new user - need help with ingestion

### DISCUSSION THREAD

**Discussion by newgit01** [datalust/seq-tickets#1301] [General]
Hello all, I am attempting to import some log files (collected from another system's syslog and downloaded locally).

When I try to set up ingestion, I am using the cmd prompt, navigating to the folder containing the log files, and enter "seqcli ingest -i* filename *.log.   The result I receive is:
Failed to send an event batch
System.Net.Http.HttpRequestException: The SSL connection could not be established, see inner exception.
 ---> System.IO.IOException: Cannot determine the frame size or a corrupted frame was received.

I have downloaded and extracted the command line client, following the written procedure.

Any help is appreciated.

**Comment by nblumhardt** [comment]
Hi 👋 

Is your Seq server listening at an HTTPS endpoint? It looks like `seqcli` is using an HTTPS server URL (e.g. `seqcli config -k connection.serverUrl -v https://some-server`) but the server isn't serving HTTPS at that location (may just need to change it to HTTP?).

Let me know if this helps - will need some more info about the server's location, the operating systems involved, the network topology, and what Seq looks like in a browser (certificate warnings?) otherwise.

Best regards,
Nick

**Comment by newgit01** [comment]
Hi Nick, please disregard the seq server feed on my account (if that is what you’re looking at).

I am trying to import files from a folder on my hard drive using seqcli ingest via a command prompt for this issue.

In a command prompt, I navigate to the folder containing the text files I wish to import.  I then use the command _seqcli ingest -i filesname*.log_ but receive the following error:

Failed to send an event batch
System.Net.Http.HttpRequestException: The SSL connection could not be established, see inner exception.
---> System.IO.IOException: Cannot determine the frame size or a corrupted frame was received.

**Reply by nblumhardt** [reply]
Thanks for your reply. I don't have any further info to go on, here - just your post above :-)

`seqcli ingest` will call Seq via its HTTP ingestion API. The source of the error will come down to a mismatch between where `seqcli` thinks the server is listening (the `connection.serverUrl` setting), and where the Seq server itself is listening (whatever URL you access Seq on in the browser).

Are these values the same?

HTH,
Nick
